src/core/services/value.d.ts

```diff
@@ -0,0 +1,8 @@
+import {ColumnModel} from '../column-type/column.model';
+import {Row} from '../row/row';
+
+export declare function get(row: Row, column: ColumnModel): any;
```

Comment:
I'm not sure about first parameter as ```Row```. It should be one of objects from ```model.data().rows``` which are collection of objects. So I think, ```row``` is ```object``` too